### PR TITLE
adds typeahead for hard coded list of chefs

### PR DESCRIPTION
**Pull request by tsop14**
- stops enter submitting form by turning off default value (might want to fine tune this) #51
- adds typeahead for chefs,  currently using hard coded list. #48

(List is unique contributor tags for 100 most recent recipe articles.)

autosuggest

### CODE REVIEW

**Comment by tsop14** [comment]
I reran ETL with byline field and took all the distinct row (with some minimal exclusion) and put into a json file to give a fuller list.

**Comment by LATaylor-guardian** [comment]
@alastairjardine Might have an opinion on this list?

**Comment by alastairjardine** [comment]
Can you use the keyboard to select the chef/cook?

**Comment by mchv** [comment]
How is the `json` file generated?

File: ui/public/javascript/credits.json [hunk 1]
```diff
@@ -0,0 +1,661 @@
...
+"Linda Tubby",
+"Preman Mohan",
+"Andy McLeish",
+"Zuza Zak",
+"Interview by Lizzie Enfield",
+"Jason Atherton",
+"Kate Percy",
+"Maureen Pogson",
+"Fraser Lewry",
+"Gérard Depardieu",
```
Comment (mchv) [inline ui/public/javascript/credits.json]: French actor and vineyard owner, very funny to spot him there!
🍷

**Comment by mchv** [comment]
👍

**Comment by tsop14** [comment]
> How is the json file generated?

It is the output of this psql query to recipe table:

`SELECT DISTINCT credit from recipe WHERE credit NOT LIKE '%,%' AND credit NOT LIKE '%.%' AND credit NOT LIKE '%;%';`

NB this will only work if credit field is populated by running updated ETL as per #76

File: ui/public/javascript/editForm.js [hunk 2]
```diff
@@ -73,7 +75,13 @@ function guessIngredient(){
 }
 
 $( document ).ready(function() {
-  guessIngredient()
+    guessIngredient()
+    $(window).keydown(function(event){
+        if(event.keyCode == 13) {
```
Comment (tsop14) [inline ui/public/javascript/editForm.js]: Done

**Pull request merged**